pkg/cmd/experimental/diagnostics/host.go

```diff
@@ -0,0 +1,76 @@
+package diagnostics
+
+import (
+	"fmt"
+	"github.com/openshift/origin/pkg/diagnostics/host"
+	systemddiagnostics "github.com/openshift/origin/pkg/diagnostics/systemd"
+	"github.com/openshift/origin/pkg/diagnostics/types"
+	"k8s.io/kubernetes/pkg/util"
+	"os"
+)
+
+const (
+	StandardMasterConfigPath string = "/etc/openshift/master/master-config.yaml"
+	StandardNodeConfigPath   string = "/etc/openshift/node/node-config.yaml"
+)
+
+var (
+	AvailableHostDiagnostics = util.NewStringSet("AnalyzeLogs", "UnitStatus", "MasterConfigCheck", "NodeConfigCheck")
+)
+
+func (o DiagnosticsOptions) buildHostDiagnostics() ([]types.Diagnostic, bool /* ok */, error) {
```

Comment:
I was never really happy about this, but I do think there is a useful distinction between indicating "there was an error we should track" vs "we should abort running any diagnostics".